Dockerfile

```diff
@@ -1,12 +1,20 @@
+FROM kbase/kb_jre:latest as dockerize
```

Comment:
If dockerize just needs to be in the path, then I wonder if it can be installed from a smaller image? It looks like the official `jwilder/dockerize` image is based on alpine. I was going to try to do that in the cache repo, assuming dockerize is required to deploy

Reply:
As I understand it (and @scanon is the expert here), this is a two stage build, so the kb_jre image doesn't end up in the final image. The only benefit from switching from kb_jre to a different image would be it'd be slightly faster to pull that image when building the idmapper image. It doesn't have any effect on the final size of the idmapper image.

Reply:
Yeah, I was only thinking of pull time -- didn't want to pull the java runtime environment just to get dockerize